README.md

```diff
@@ -1,427 +1,96 @@
+   - [Design Challenge: Sybil Attacks](./articles/design-challenge-sybil-attack.md)
+   - [Desirable Properties for a Secure Channel](./articles/desirable-properties-for-a-secure-channel.md)
+   - [Secure, Private Channels: the Good, the Bad, and the Ugly](./articles/secure-private-channels.md)
+ - Scuttlebot
+   - [Repo & Overview](https://github.com/ssbc/scuttlebot)
+   - [An Introduction to Developing with Scuttlebot](todo)
+   - [API Docs](https://github.com/ssbc/scuttlebot/blob/master/api.md)
+   - Plugins
+     - [Blobs](https://github.com/ssbc/scuttlebot/blob/master/plugins/blobs.md)
+     - [Block](https://github.com/ssbc/scuttlebot/blob/master/plugins/block.md)
+     - [Friends](https://github.com/ssbc/scuttlebot/blob/master/plugins/friends.md)
+     - [Gossip](https://github.com/ssbc/scuttlebot/blob/master/plugins/gossip.md)
+     - [Invite](https://github.com/ssbc/scuttlebot/blob/master/plugins/invite.md)
+     - [Private](https://github.com/ssbc/scuttlebot/blob/master/plugins/private.md)
+     - [Replicate](https://github.com/ssbc/scuttlebot/blob/master/plugins/replicate.md)
+ - Pull-streams
+   - [Repo & Overview](https://github.com/dominictarr/pull-stream)
+   - [A Primer for Pull-streams](todo)
+   - [Pull Sources](https://github.com/dominictarr/pull-stream/blob/master/docs/sources.md)
+   - [Pull Throughs](https://github.com/dominictarr/pull-stream/blob/master/docs/throughs.md)
+   - [Pull Sinks](https://github.com/dominictarr/pull-stream/blob/master/docs/sinks.md)
+
+
+## Glossary
+
+ - [Secure-Scuttlebutt](https://github.com/ssbc/secure-scuttlebutt) (SSB) - A protocol for replicating logs in a decentralized mesh. The SSB library wraps leveldb with tools for reading, writing to, and replicating feeds.
+ - [Scuttlebot](https://github.com/ssbc/scuttlebot) - A secure-scuttlebutt server and client. Includes the database, networking, blob exchange protocol, and command-line tools.
+ - Pub Servers - SSB peers which run on publicly IPs, and provide connectivity and hosting for users on private IPs. Pubs are not privileged, and do not hold special authority in the network. They are not hosts.
+ - Invite codes - Tokens which may be used to command specific Pub servers to follow a user. These are used to join Pubs.
+ - Feeds - a user's stream of JSON data. Also called logs.
+ - Gossip - a networking technique where peers connect randomly to each other and ask for new updates. Gossip is unique for not relying on any central server to push information. Instead, the information travels peer-to-peer.
 
 
 ## Overview
 
-Secure Scuttlebutt is a global mesh database.
-It uses a trustless protocol, meaning any device can participate in the mesh.
-It is eventually-consistent, meaning devices can freely enter and exit the network without stopping progress.
+Scuttlebot is a mesh database for JSON logs.
+It uses a trustless protocol, so any device can participate in the mesh.
+It's used with desktop applications to publish and syncronize user-data.
```

Comment:
I think @ahdinosaur has a good point here. Though, you could make a case that the protocol is trustless, but that the applications are often _about_ trust. The idea is that you don't have to trust computers (except your own) but that you can your friends. However, this is a sublty that we probably don't need to get to just now.